### DISCUSSION TITLE
Widget: New Data source

### DISCUSSION THREAD

**Discussion by LimitlessDrag0n** [gameserverapp/Platform#497] [Suggestions]
![image](https://github.com/gameserverapp/Platform/assets/99832504/aaa8e022-0005-480f-89e0-ee421815a919)

Maybe there is an option for getting "new player" instead of/or/and "new characters" because one player can create multiple characters and I think, that can be misleading or confusing.

**Comment by LimitlessDrag0n** [comment]
Addition: A year is cool, but after a year, what's then? After a year would be nice to have: "all time"

**Reply by mvaessen** [reply]
hi, for new suggestions please add a new suggestion. In the case of longer time frames: this unfortunately isn't something i consider as loading `all` data would have a significant impact on the platforms performance. The limit of a year is basically a security measure to protect the platform from very intense queries.
